.nix/shellHook.sh

```diff
@@ -159,8 +159,11 @@ addNixCommand ppNixAction
 genNixActions (){
   mkdir -p $currentDir/.github/workflows/
   for t in $bundles; do
-    echo "generating $currentDir/.github/workflows/nix-action-$t.yml"
-    nix-shell --arg do-nothing true --argstr bundle $t --run "ppNixAction > $currentDir/.github/workflows/nix-action-$t.yml"
+    for p in ubuntu macos; do
+      echo "generating $currentDir/.github/workflows/nix-action-$t-$p.yml"
+      nix-shell --arg do-nothing true --argstr bundle $t --run "ppNixAction > $currentDir/.github/workflows/nix-action-$t-$p.yml"
+      sed -i "s/ubuntu/$p/" "$currentDir/.github/workflows/nix-action-$t-$p.yml"
```

Comment:
Could we rather change the nix script called by `ppNixAction`?

Reply:
This PR was meant more as a POC than an actual proposal. Should we add a new `platform` argument to `default.nix`? Do you see another way?